src/schemapack/_internals/cli/main.py

```diff
@@ -117,3 +119,53 @@ def check_datapack(
         load_datapack(datapack)
 
     print_final_success("The provided document complies with the specs of a datapack.")
+
+
+@cli.command()
+def export_mermaid(
+    *,
+    schemapack: Annotated[
+        Path,
+        typer.Argument(
+            help="Provide the path to a schemapack file to export.",
+        ),
+    ],
+    output: Annotated[
+        Path,
+        typer.Argument(
+            help="Provide the path to the mermaid output file to produce.",
+            file_okay=True,
+            dir_okay=False,
+            writable=True,
+        ),
+    ],
+    with_properties: Annotated[
+        bool,
+        typer.Option(
+            "--with-properties",
+            "-p",
+            help="Include properties in the output.",
+        ),
+    ] = False,
+    force: Annotated[
+        bool,
+        typer.Option(
+            "--force",
+            "-f",
```

Comment:
I would prefer to keep this unchanged, since '-f / --force' is a pretty well established flag for descructive behaviours, most popularly overwriting outputs.

Reply:
Alright, no objections.